src/components/behandlerdialog/meldingtilbehandler/MeldingTilBehandlerSkjema.tsx

```diff
@@ -114,26 +114,28 @@ export const MeldingTilBehandlerSkjema = () => {
           {`Meldingen ble sendt ${tilDatoMedManedNavnOgKlokkeslett(now)}`}
         </Alert>
       )}
-      <Select
-        id="type"
-        className={"w-[23rem]"}
-        label={texts.meldingsType.label}
-        {...register("meldingsType", { required: true })}
-        value={watch("meldingsType")}
-        error={errors.meldingsType && texts.meldingsType.missing}
-      >
-        <option value="">{texts.meldingsType.defaultOption}</option>
-        <MeldingTypeOption
-          type={MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER}
-        />
-        <MeldingTypeOption
-          type={MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING}
-        />
-        <MeldingTypeOption type={MeldingType.HENVENDELSE_MELDING_FRA_NAV} />
-      </Select>
-      {watch("meldingsType") && (
-        <MeldingsTypeInfo meldingType={watch("meldingsType")} />
-      )}
+      <div className="max-w-[23rem]">
+        <Select
+          id="type"
+          className="mb-4"
+          label={texts.meldingsType.label}
+          {...register("meldingsType", { required: true })}
+          value={watch("meldingsType")}
+          error={errors.meldingsType && texts.meldingsType.missing}
+        >
+          <option value="">{texts.meldingsType.defaultOption}</option>
+          <MeldingTypeOption
+            type={MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER}
+          />
+          <MeldingTypeOption
+            type={MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING}
+          />
+          <MeldingTypeOption type={MeldingType.HENVENDELSE_MELDING_FRA_NAV} />
+        </Select>
+        {watch("meldingsType") && (
+          <MeldingsTypeInfo meldingType={watch("meldingsType")} />
+        )}
+      </div>
```

Comment:
Trenger vi `div` som wrapper her?
```suggestion
        <Select
          id="type"
          className="mb-4 max-w-[23rem]"
          label={texts.meldingsType.label}
          {...register("meldingsType", { required: true })}
          value={watch("meldingsType")}
          error={errors.meldingsType && texts.meldingsType.missing}
        >
          <option value="">{texts.meldingsType.defaultOption}</option>
          <MeldingTypeOption
            type={MeldingType.FORESPORSEL_PASIENT_TILLEGGSOPPLYSNINGER}
          />
          <MeldingTypeOption
            type={MeldingType.FORESPORSEL_PASIENT_LEGEERKLARING}
          />
          <MeldingTypeOption type={MeldingType.HENVENDELSE_MELDING_FRA_NAV} />
        </Select>
        {watch("meldingsType") && (
          <MeldingsTypeInfo meldingType={watch("meldingsType")} />
        )}
```

Reply:
Var jo sånn det var før, og tanken med wrapperen er å samle `MeldingTypeInfo` inn under samme bredde som selecten. Jeg tenker de hører litt sammen ettersom den ene er et resultat av hva du setter som verdi i Selecten. Alternativet er å ha `max-w-[23rem]` på både Select og MeldingTypeOption, men hvis man da i fremtiden skulle endre på den ene, må man også huske på å endre den andre.